api/v1alpha1/wasm_types.go

```diff
@@ -115,6 +115,11 @@ type HTTPWasmCodeSource struct {
 	// kubebuilder:validation:Pattern=`^[a-f0-9]{64}$`
 	// +optional
 	SHA256 *string `json:"sha256"`
+
```

Comment:
should this also use BackendCluster ?

Reply:
It's possible, but using BackendCluster may not be the appropriate approach - the TLS setting are used by the control plane rather than the data plane.

Reply:
ah makes sense